### ISSUE TITLE
Function name contains drupal.

### ISSUE THREAD

**Issue by Graham-72** [backdrop-contrib/colorbox#4]
Just need to make it `backdrop_ucfirst(` in [colorbox.install line 17](https://github.com/backdrop-contrib/colorbox/blob/1.x-1.x/colorbox.install#L17).

**Comment by Graham-72** [comment]
I have made a pull request https://github.com/backdrop-contrib/colorbox/pull/5 to fix this.

**Comment by laryn** [comment]
This just crashed my site when I was trying to install the module. @Graham-72 as it appears there is no maintainer, what's the process for getting this merged?

**Comment by Graham-72** [comment]
@laryn I'm sorry to hear of the problem. I am a user of Colorbox and feel it would be worthwhile dealing with any outstanding issues.

@biolithic would you be happy for me to do a bit of maintenance and produce a new release?

**Comment by jenlampton** [comment]
@laryn @Graham-72 the process for abandoned projects is documented here: https://github.com/backdrop-ops/contrib#abandoned-projects

**Comment by klonos** [comment]
@jenlampton what about that "contrib squad" that we were talking about taking action? I mean:

1. we know for sure that @biolithic (the original porter) has not been active lately.
2. this is a simple, 1-liner change.
3. it is obvious that this is a change that needs to be made if the port is to work with Backdrop.
4. @laryn reports that without the fix, there are major severity consequences on sites using the module.

...and in this specific case, we clearly have 3 people involved in this issue that are already active (and may I add trustworthy) members of the backdrop-contrib group.

The bottom line is that none of us might ever want to adopt this specific abandoned project, but we have a simple, obvious fix that is RTBC'ed. I think we should merge and create a new release.

**Comment by jenlampton** [comment]
> what about that "contrib squad" that we were talking about taking action? 

Well, it doesn't exist yet, so we can't use it yet. I think it's a good idea and we should start moving that way, but it doesn't apply here...  since it doesn't exist :)

> ...and in this specific case, we clearly have 3 people involved in this issue that are already active (and may I add trustworthy) members of the backdrop-contrib group.

So in this specific case one of them (I'm happy to do it if no one else is) should adopt the project, merge the fix, and create a new release.

> The bottom line is that none of us might ever want to adopt this specific abandoned project

If that's the case, then we use the project as-is, apply the RTBC's patch, and carry on. Just as we would with any other open source project that doesn't have a bug squad.

**Comment by klonos** [comment]
> I'm happy to do it if no one else is

Yes please 😄 

> If that's the case, then we use the project as-is, apply the RTBC's patch, and carry on. Just as we would with any other open source project that doesn't have a bug squad.

Fair enough. Was just pointing the fact that this is a simple fix that solves a major issue (installing the project without having the entire site go bananas), which affects all users of the current official release of the project. But fair enough.

**Comment by jenlampton** [comment]
> Yes please 😄

okiedokie, I've created the first issue: https://github.com/backdrop-contrib/colorbox/issues/11  Two weeks from now, if he hasn't replied, I can create an issue in the contrib queue to take over the project!

> Fair enough. Was just pointing the fact that this is a simple fix that solves a major issue (installing the project without having the entire site go bananas), which affects all users of the current official release of the project. But fair enough.

I agree with all those things. I just think we should follow our own rules. If you want to start work on a bug squad (to change those rules) this is a great issue to use as an example.

**Comment by klonos** [comment]
> I just think we should follow our own rules.

Okay 👿

**Comment by jenlampton** [comment]
This issue was resolved by #9.
